### ISSUE TITLE
Move annotations and GWAS data to the LIS data store, then remove local versions

### ISSUE THREAD

**Issue by svengato** [legumeinfo/ZBrowse#21]
From the annotations and www/config/data subdirectories respectively.

**Comment by svengato** [comment]
Are pigeonpea GWAS data (in the same format as common bean, cowpea, and soybean, including marker and publication) ready to go on the data store?

**Comment by adf-ncgr** [comment]
no, I think this is likely to languish for a little while, but as a note to self these were (I think) taken from the publication:
https://doi.org/10.1038/ng.3872
probably from Supp. Table 18 which doesn't actually give p-values, but says that the indicated marker-trait associations are all p-value < 0.05
